src/pages/[platform]/build-a-backend/storage/download-files/index.mdx

```diff
@@ -0,0 +1,63 @@
+import { Card } from '@aws-amplify/ui-react';
+import { getCustomStaticPath } from '@/utils/getCustomStaticPath';
+
+export const meta = {
+  title: 'Download files',
+  description: 'Download files using Amplify Storage',
+  platforms: [
+    'angular',
+    'javascript',
+    'nextjs',
+    'react',
+    'vue'
+  ]
+};
+
+export const getStaticPaths = async () => {
+  return getCustomStaticPath(meta.platforms);
+};
+
+export function getStaticProps(context) {
+  return {
+    props: {
+      meta
+    }
+  };
+}
+
+You can easily display images on your app by using the cloud-connected Storage Image UI component. This component fetches images securely from your storage resource and displays it on the web page.
+
+```
+npm install @aws-amplify/ui-react-storage aws-amplify
+```
+```
+import { StorageImage } from '@aws-amplify/ui-react-storage';
+
+export const DefaultStorageImageExample = () => {
+  return <StorageImage alt="cat" imgKey="cat.jpg" accessLevel="guest" />;
+};
+```
```

Comment:
the suggestion doesn't want to take because of the code blocks 😅 

````
To get started, install the relevant UI library package
```bash
npm install @aws-amplify/ui-react-storage
```

Then you can import the `StorageImage` component and render an image:

```tsx
import { StorageImage } from '@aws-amplify/ui-react-storage';

export const DefaultStorageImageExample = () => {
  return <StorageImage alt="cat" imgKey="cat.jpg" accessLevel="guest" />;
};
```
````